tests/unit_test/test_local_video.py

```diff
@@ -21,6 +21,7 @@ def test_local_video_regular_function():
     id_category = 1
     privacy_statuses = ["public", "private", "unlisted"]
     playlist_id = "some_playlist_id"
+    publish_at = datetime.datetime(2028, 5, 17)
```

Comment:
This will fail in 2028(lol), can you make the time like one year more than current year so it never fails.